frontend/css/fresh-releases.less

```diff
@@ -358,3 +439,123 @@
   justify-content: center;
   flex-direction: column;
 }
+
+.toggle-sidebar-button {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  z-index: 9999;
+  width: 64px;
+  height: 64px;
+  padding: 10px;
+  border-radius: 50%;
+  border-color: black;
+  cursor: pointer;
+  background-color: #fff;
+  color: #353070;
+
+  @media screen and (min-width: @screen-md) {
+    display: none;
+  }
+}
+
+.toggle-sidebar-button.open {
+  background-color: #353070;
+  color: #fff;
+}
+
+.no-release {
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+
+  img {
+    width: 350px;
+    height: 350px;
+  }
+}
+
+#sidebar-fresh-release {
+  background: #353070;
+  color: white;
+  padding-top: 20px;
+
+  p:first-of-type {
+    font-size: 25px;
+    font-weight: 900;
+  }
+
+  p {
+    margin-bottom: 5px;
+  }
+}
+
+.releases-page-container {
+  display: flex;
+  gap: 1.5em;
+  @media screen and (min-width: @screen-md) {
+    // make the sidebar flush with the right-hand side on larger sizes
+    margin-right: -15px;
+  }
+  .releases-page {
+    flex: 1;
+  }
+  .sidebar {
+    flex: 0 0 250px;
+  }
+}
+
+.sidebar {
+  position: sticky;
+  top: 10px;
+  border-radius: 10px;
+  height: max-content;
+  max-height: 100vh;
+  overflow-y: scroll;
+  z-index: 9998;
+  right: 0;
+
+  @media screen and (max-width: @screen-md) {
+    position: fixed;
+    top: 0;
+    right: -80%;
+    height: 100%;
+    width: 80%;
+    max-width: 400px;
+    background-color: #fff;
+    overflow-y: auto;
+    box-shadow: -2px 0px 5px rgba(0, 0, 0, 0.2);
+    transition: right 0.3s ease-in-out;
+    &.open {
+      right: 0;
+    }
+  }
+
+  .release-tags {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+  }
+
+  .tags {
+    display: inline-block;
+    margin: 2px 2px 5px;
+    padding: 3px;
+  }
+
+  .release-tag-name {
+    margin-right: 5px;
+    font-weight: 500;
+    font-size: 13px;
+  }
+}
+
+.sidebar-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(255, 255, 255, 0.8);
+  z-index: 1;
```

Comment:
We need to add a `border: none;` as well, otherwise the standard styling of the `<button>` element applies a black border:
<img width="249" alt="image" src="https://github.com/metabrainz/listenbrainz-server/assets/6179856/0904182c-7a9b-4f69-825f-cfd62461dc5f">


```suggestion
  z-index: 1;
  border: none;
```